### PR TITLE
Add Indonesian translation

### PR DESCRIPTION
**Pull request by xaliazhar**

### CODE REVIEW

**Comment by aliazhar-id** [comment]
#149

**Comment by amd64fox** [comment]
- please return `[Y/N]` instead of `[Y/T]`

- you also need to add the author of the translation, i.e. yourself:
you need to add these lines after the patch author
```ps1
TranslationBy   = "Translation author:"
TranslationBy2  = "@"
```

File: scripts/installer-lang/id.ps1 [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    NoVariable5     = "di home-v2.js"
+    NoVariable6     = "di xpui-desktop-modals.js"
+    ModSpoti        = "Menambal Spotify..."
+    Error           = "Kesalahan"
+    FileLocBroken   = "Lokasi file Spotify rusak, hapus instalan klien dan jalankan skrip lagi"
+    Spicetify       = "Spicetify terdeteksi"
+    NoRestore       = "SpotX telah diinstal, tetapi file untuk memulihkan xpui.js.bak dan xpui.css.bak tidak ditemukan. `nHarap hapus instalan klien Spotify dan jalankan Install.bat lagi"
+    ExpSpotify      = "Fitur eksperimental yang dioperasikan oleh Spotify"
+    ExpStandart     = "Fitur eksperimental SpotX tidak termasuk"
+    NoRestore2      = "SpotX sudah terinstal, xpui.bak tidak ditemukan. Harap hapus instalan klien Spotify dan jalankan Install.bat lagi"
```
Comment (amd64fox) [inline scripts/installer-lang/id.ps1]: Lost `n at line 56

File: scripts/installer-lang/id.ps1 [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    Author          = "Penulis tambalan:"
+    Author2         = "@Amd64fox"
+    Incorrect       = "Ups, nilai yang salah,"
+    Incorrect2      = "masukkan lagi"
+    Download        = "Kesalahan saat mengunduh"
+    Download2       = "Akan memuat ulang dalam 5 detik..."
+    Download3       = "Kesalahan lagi"
+    Download4       = "Periksa pengaturan jaringan/internet anda dan jalankan instalasi lagi"
+    Download5       = "Mengunduh Spotify"
+    Download6       = "Curl error"
```
Comment (amd64fox) [inline scripts/installer-lang/id.ps1]: not translated in line 11 ?

Comment (xaliazhar) [reply scripts/installer-lang/id.ps1]: ok i will fix it

File: scripts/installer-lang/id.ps1 (hunk 1 above)
Comment (aliazhar-id) [reply scripts/installer-lang/id.ps1:56]: @amd64fox isn't "`n" just not there?
only on line 53

**Comment by xaliazhar** [comment]
Done!

**Comment by amd64fox** [comment]
merged, thank you very much

**Pull request merged**